Documentation/telemetry/telemeter_query

```diff
@@ -1 +1 @@
_error_count_total|log_forwarder_pipeline_info|log_forwarder_input_info|log_forwarder_output_info|cluster:log_collected_bytes_total:sum|cluster:log_logged_bytes_total:sum|openshift_logging:log_forwarder_pipelines:sum|openshift_logging:log_forwarders:sum|openshift_logging:log_forwarder_input_type:sum|openshift_logging:log_forwarder_output_type:sum|openshift_logging:vector_component_received_bytes_total:rate5m|cluster:kata_monitor_running_shim_count:sum|platform:hypershift_hostedclusters:max|platform:hypershift_nodepools:max|cluster_name:hypershift_nodepools_size:sum|cluster_name:hypershift_nodepools_available_replicas:sum|namespace:noobaa_unhealthy_bucket_claims:max|namespace:noobaa_buckets_claims:max|namespace:noobaa_unhealthy_namespace_resources:max|namespace:noobaa_namespace_resources:max|namespace:noobaa_unhealthy_namespace_buckets:max|namespace:noobaa_namespace_buckets:max|namespace:noobaa_accounts:max|namespace:noobaa_usage:max|namespace:noobaa_system_health_status:max|ocs_advanced_feature_usage|os_image_url_override:sum|cluster:vsphere_topology_tags:max|cluster:vsphere_infrastructure_failure_domains:max|apiserver_list_watch_request_success_total:rate:sum|rhacs:telemetry:rox_central_info|rhacs:telemetry:rox_central_secured_clusters|rhacs:telemetry:rox_central_secured_nodes|rhacs:telemetry:rox_central_secured_vcpus|rhacs:telemetry:rox_sensor_info|cluster:volume_manager_selinux_pod_context_mismatch_total|cluster:volume_manager_selinux_volume_context_mismatch_warnings_total|cluster:volume_manager_selinux_volume_context_mismatch_errors_total|cluster:volume_manager_selinux_volumes_admitted_total|ols:provider_model_configuration|ols:rest_api_query_calls_total:2xx|ols:rest_api_query_calls_total:4xx|ols:rest_api_query_calls_total:5xx|openshift:openshift_network_operator_ipsec_state:info|cluster:health:group_severity:count",action=~"Pass|Allow|Deny|Allow|Deny|",alertstate=~"firing|",direction=~"Ingress|Egress|Ingress|Egress|",enabled=~"true|false|",page=~"overview-classic|overview-fleet|search|search-details|clusters|application|governance|",quantile=~"0.99|0.99|0.99|",reason=~"memory_working_set_delta_from_request|memory_rss_delta_from_request|",severity=~"critical|warning|info|none|critical|warning|info|none|",state=~"Managed|Unmanaged|",system_type=~"OCS|OCS|",system_vendor=~"Red Hat|Red Hat|",table_name=~"ACL|Address_Set|ACL|Address_Set|",type=~"azure|gcs|s3|enabled|disabled|jaeger|hostmetrics|opencensus|prometheus|zipkin|kafka|filelog|journald|k8sevents|kubeletstats|k8scluster|k8sobjects|otlp|debug|logging|otlp|otlphttp|prometheus|lokiexporter|kafka|awscloudwatchlogs|loadbalancing|batch|memorylimiter|attributes|resource|span|k8sattributes|resourcedetection|filter|routing|cumulativetodelta|groupbyattrs|zpages|ballast|memorylimiter|jaegerremotesampling|healthcheck|pprof|oauth2clientauth|oidcauth|bearertokenauth|filestorage|spanmetrics|forward|deployment|daemonset|sidecar|statefulset|",vendor=~"NVIDIA|AMD|GAUDI|INTEL|QUALCOMM|",verb=~"LIST|WATCH|"}
```

Comment:
below type appears twice
```
kafka
memorylimiter
otlp
prometheus
```

Reply:
Yeah I think that is expected. Those terms appear as label values in multiple queries and our processing doesn't deduplicate.